js/util/browser/browser.js

```diff
@@ -2,6 +2,27 @@
 
 var Canvas = require('./canvas');
 
+/*
+ * Unlike js/util/browser.js, this code is written with the expectation
+ * of a browser environment with a global 'window' object
+ */
+
+/**
+ * Provides a function that outputs milliseconds: either performance.now()
+ * or a fallback to Date.now()
+ * @private
+ */
+var nowFn = (function() {
+    if (window.performance &&
+        window.performance.now) {
+        return window.performance.now.bind(window.performance);
+    } else {
+        return Date.now.bind(Date);
+    }
+}());
+
+module.exports.nowFn = nowFn;
```

Comment:
We've been [calling its output now](https://github.com/mapbox/mapbox-gl-js/pull/2057/files#diff-e60957272552aeccdb45f65d82d523e1L47). Unless we rename the output, we shouldn't call this now.